xds/src/main/java/io/grpc/xds/LoadReportClient.java

```diff
@@ -357,6 +364,37 @@ void sendLoadStatsRequest(List<ClusterStats> clusterStatsList) {
     void sendError(Exception error) {
       lrsRequestWriterV2.onError(error);
     }
+
+    private io.envoyproxy.envoy.api.v2.endpoint.ClusterStats buildClusterStats(
```

Comment:
These are the original `toEnvoyProto()`. Put them here where they are actually used. They are effectively static helper methods. Not static just because it is put in the inner class. If you really want them to be static, I can put them in `LoadReportClient`.